tests/test_bindings.py

```diff
@@ -244,3 +244,28 @@ def test_box_seal_wrong_types():
     with pytest.raises(TypeError):
         c.crypto_box_seal_open(
             None, A_pubkey, A_secretkey)
+
+
+def test_box_seed_keypair_reference():
+
+    from box_secret_from_seed_ref import test_seed, test_pk, test_sk
```

Comment:
I don't think these need to be in a separate file, let's just have them be inline in this file along with the comments.

Reply:
Oh, having said that I see they're used in both test_bindings and test_box so nevermind.

Do we have any other python files containing vectors like this? Typically we've made these types of vectors text files that we load and parse.

Reply:
I think I've begun preparing this PR before #297 got merged, and I didn't want to add another open-coded version of test.utils.read_crypto_test_vectors, which would have been necessary to add the needed comments about the reference data. Hope this is better now.

Reply:
OK rebased on master after merge of #316